cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -19,26 +19,18 @@
         role="banner">
 
     {% if flag_enabled('BETA_NOTICE', request) and show_banner %}
+    {% from 'molecules/notification.html' import render as notification with context %}
```

Comment:
I don't think we should be renaming functions like this, why not

```
{% import 'molecules/notification.html' as notification with context %}
```

And call `notification.render(…)` elsewhere.

Reply:
Good call. I copied this from elsewhere, I'll be sure to update it there as well.